assets/js/components/GoogleChart.js

```diff
@@ -185,6 +197,23 @@ export default function GoogleChart( props ) {
 		} );
 	}
 
+	if ( zeroDataStatesEnabled && gatheringData && chartType === 'LineChart' ) {
```

Comment:
I think we need to create a copy of the `options` property to make sure that our changes here don't modify the original object passed from the parent component:

```js
import cloneDeep from 'lodash/cloneDeep';

let chartOptions = cloneDeep( options );
if ( ... ) {
    set( chartOptions, ... );
}

<Chart ... options={ chartOptions } ... />
```